package.json

```diff
@@ -77,16 +79,14 @@
     "@types/lodash.property": "^4.4.7",
     "@types/lodash.set": "^4.3.7",
     "@types/lodash.startcase": "^4.4.7",
-    "@types/node": "^18.16.19",
-    "@types/react": "^18.2.15",
+    "@types/node": "^18.17.1",
```

Comment:
I think we probably just want a hard `18.16.1` here as it's tied to our Node version. This is something we need to look at in the other repos also.